src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
+        this.line = line;
+        this.comment = comment;
+    }
+
+    public int getLine() {
+        return line;
+    }
```

Comment:
:information_source: The field 'okbb' is not following Java naming conventions. It should start with a lowercase letter. (Severity: Minor)